### PR TITLE
[6.14.z] Cherrypick test sync multiple large repos 6.14.z

### PR DESCRIPTION
**Pull request by ColeHiggins2**
ParentPR: https://github.com/SatelliteQE/robottelo/pull/12445
issue: https://github.com/SatelliteQE/robottelo/issues/12742

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_repositories.py -k test_positive_sync_mulitple_large_repos

**Comment by Gauravtalreja1** [comment]
@ColeHiggins2 We already had the cherrypicked PR https://github.com/SatelliteQE/robottelo/pull/12741 for this, this issue must be created due to intermittent memory issue, so  would suggest to check the original PR where cherrypicked PRs and created issues will be linked, so before manually cherrypicking a PR we should be verifying if it is not already cherrypicked in some PR :)
Will close PR https://github.com/SatelliteQE/robottelo/pull/12741 for now

**Review by lhellebr — CHANGES_REQUESTED** [review]
Logicx of this seems ok.

There are a few comments from me pointing to either a bad naming or bad understanding of what's actually happening - you should think about it and rename accordingly.

Also, think about putting this test to `longrun` because, well, it may run long.

**Comment by Gauravtalreja1** [comment]
@ColeHiggins2 Can you rebase and run pre-commit locally? and take a look at the @lhellebr's comments

**Comment by adarshdubey-star** [comment]
Ping @ColeHiggins2

File: tests/foreman/api/test_repositories.py [hunk 1]
```diff
@@ -243,3 +245,51 @@ def test_positive_multiple_orgs_with_same_repo(target_sat):
...
+        3. Assert that the bulk sync succeeds
+
+    :expectedresults: All repositories should sync with no errors
+
+    :BZ: 2224031
+    """
+    repo_names = ['rhel8_bos', 'rhel8_aps']
+    kickstart_names = ['rhel8_bos', 'rhel8_aps']
+    for name in repo_names:
+        rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
```
[the remark this answers is not in the document]
Comment (ColeHiggins2) [reply tests/foreman/api/test_repositories.py]: I can remove this var and see if it passes prt

File: tests/foreman/api/test_repositories.py [hunk 2]
```diff
@@ -243,3 +245,51 @@ def test_positive_multiple_orgs_with_same_repo(target_sat):
...
+            basearch=constants.DEFAULT_ARCHITECTURE,
+            org_id=module_entitlement_manifest_org.id,
+            product=constants.REPOS['kickstart'][name]['product'],
+            repo=constants.REPOS['kickstart'][name]['name'],
+            reposet=constants.REPOS['kickstart'][name]['reposet'],
+            releasever=constants.REPOS['kickstart'][name]['version'],
+        )
+    rh_repos = module_target_sat.api.Repository(id=rh_repo_id).read()
+    rh_products = module_target_sat.api.Product(id=rh_repos.product.id).read()
```
[the remark this answers is not in the document]
Comment (ColeHiggins2) [reply tests/foreman/api/test_repositories.py]: updating to singular

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_repositories.py -k test_positive_sync_mulitple_large_repos

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**